docs/3.0/customization.md

```diff
@@ -228,20 +228,52 @@ Have the same output:
 create  app/components/avo/index/table_row_component.rb
 create  app/components/avo/index/table_row_component.html.erb
 ```
+:::
+
+:::option `--field-components`
```

Comment:
Let's link to the field-options path somewhere below this so they can reach it more easily.